insights/tests/parsers/test_jboss_version.py

```diff
@@ -34,11 +34,55 @@ def test_jboss7():
     assert release7.release == 0
 
 
+SINGLE_JBOSS = """
+{"/opt/jboss-datagrid-7.3.0-server": "Red Hat Data Grid - Version 7.3.0"}
+"""
+
+MULTI_JBOSS = """
+{"/opt/jboss": "Red Hat JBoss Enterprise Application Platform - Version 7.3.0.GA\n", "/opt/jboss-sss": "Red Hat Single Sign-On - Version 7.4.10.GA\n", "/opt/jboss-keycloak": "Keycloak - Version 3.4.3.Final\n"}
+"""
+WEB_SERVER_RUNTIME = """
+{"/opt/jboss-web-server": "Red Hat JBoss Web Server - Version 5.6 GA\n\nJava Components:\nApache Tomcat 9.0.50-3.redhat_00004.1.el7jws\nTomcat Vault 1.1.8-4.Final_redhat_00004.1.el7jws\nJBoss mod_cluster 1.4.3-2.Final_redhat_00002.1.el7jws\n\nNative Components:\nApache Tomcat Native 1.2.30-3.redhat_3.el7jws\\nApache Portable Runtime (APR) 1.6.3-107.jbcs.el7\nOpenSSL 1.1.1g-8.jbcs.el7\n"}
+"""
+
+
+def test_JbossRuntimeVersions():
+    jboss_runtimes = JbossRuntimeVersions(context_wrap(SINGLE_JBOSS))
+    jboss_v = jboss_runtimes.versions[0]
+    assert jboss_v.file_path == "/opt/jboss-datagrid-7.3.0-server"
+    assert jboss_v.parsed['product'] == "Red Hat Data Grid"
+    assert jboss_v.version == "7.3.0"
+    assert jboss_v.major == 7
+    assert jboss_v.minor == 3
+    assert jboss_v.release == 0
+    jboss_runtimes = JbossRuntimeVersions(context_wrap(MULTI_JBOSS))
+    jboss_v = jboss_runtimes.versions[0]
+    assert jboss_v.file_path == "/opt/jboss"
+    assert jboss_v.version == "7.3.0"
+    assert jboss_v.code_name == "GA"
+    jboss_v = jboss_runtimes.versions[2]
+    assert jboss_v.file_path == "/opt/jboss-keycloak"
+    assert jboss_v.version == "3.4.3"
+    assert jboss_v.code_name == "Final"
+    jboss_runtimes = JbossRuntimeVersions(context_wrap(WEB_SERVER_RUNTIME))
+    jboss_v = jboss_runtimes.versions[0]
+    assert jboss_v.parsed['product'] == "Red Hat JBoss Web Server"
+    assert jboss_v.file_path == "/opt/jboss-web-server"
+    assert jboss_v.code_name == "GA"
+    assert jboss_v.version == "5.6"
+    assert jboss_v.major == 5
+    assert jboss_v.minor == 6
+    assert jboss_v.release == 0
+
+
 def test_jboss_version_doc_examples():
     env = {
         'JbossVersion': JbossVersion,
         'jboss_version': JbossVersion(context_wrap(JBOSS_6,
-                                                   path='/home/test/jboss/jboss-eap-6.4/version.txt'))
+                                                   path='/home/test/jboss/jboss-eap-6.4/version.txt')),
+        'JbossRuntimeVersions': JbossRuntimeVersions,
```

Comment:
This line is not required. The same to L80

Reply:
Remove them